src/test/java/org/folio/rest/impl/AccountsAPITest.java

```diff
@@ -300,6 +303,62 @@ public void shouldNotAddLoanIdIfItIsNotValidInAccount() {
     assertFalse(eventPayload.containsKey("loanId"));
   }
 
+  @Test
+  public void payCheckAmountShouldBeAllowed() {
+    Account accountToPost = postAccount();
+    ResourceClient accountsPayCheckClient = accountsPayCheckClient(accountToPost.getId());
+    AccountsCheckRequest accountCheckRequest = new AccountsCheckRequest();
+    accountCheckRequest.setAmount(3.0);
+
+    accountsPayCheckClient.attemptCreate(accountCheckRequest)
+      .then()
+      .statusCode(HttpStatus.SC_OK)
+      .body("allowed", is(true))
+      .body("amount", is(3.0f))
+      .body("remainingAmount", is(4.55f));
```

Comment:
Use value from `accountToPost` instead of literal value

Reply:
done